frontend/packages/console-shared/src/utils/viewport-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect, useCallback } from 'react';
+
+export const useViewportSize = (): { width?: number; height?: number } => {
```

Comment:
I don't think you need this if you use `DraggableCore` and better control the heights with css.